{"name": "Тест: запрос с несуществующим заказом возвращает ошибку", "status": "passed", "attachments": [{"name": "log", "source": "0752de13-46ff-4483-ad42-c2de68b24123-attachment.txt", "type": "text/plain"}], "start": 1717768238049, "stop": 1717768238351, "uuid": "1b1ae5c5-adf7-4aa3-be37-759e9d17e05d", "historyId": "4df8ca3f1fcfc736c88db5a30124b814", "testCaseId": "4df8ca3f1fcfc736c88db5a30124b814", "fullName": "tests.test_get_order.TestGetOrder#test_get_order_with_non_existent_number_order_code_400", "labels": [{"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_get_order"}, {"name": "subSuite", "value": "TestGetOrder"}, {"name": "host", "value": "DESKTOP-USF6L0H"}, {"name": "thread", "value": "8512-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_get_order"}]}